### ISSUE TITLE
Android - Paused playback UI fixes

### ISSUE THREAD

**Issue by nt294** [Stremio/stremio-features#841]
**Stremio Version**
Android 1.6.12

**Is your feature request related to a problem? Please describe.**
It is very frustrating that you cannot manually close the playback UI similar to nearly every other app, i.e. tapping a part of the screen that is not a button, and how easy it is to trigger the rewind and fast-forward buttons. 

![stremio_problem](https://github.com/Stremio/stremio-features/assets/79095109/8c165b2e-63fb-4289-9d19-2a0e20be03a6)

The picture above shows roughly the touch area for the buttons; the -15s and +15s touch areas take up far more space than they need to and often leads to accidental presses and unintentional rewinding/fast-forwarding. 

Pressing the areas not highlighted currently do nothing, and you have to wait for the UI to close. 

**Describe the solution you'd like**
![stremio_fix](https://github.com/Stremio/stremio-features/assets/79095109/27866856-c29b-4665-96a5-d84574aa2318)

1. Reduce the touch area of the +15s and -15s buttons so that they only trigger when it's clear the user has pressed them. 
2. Make it so pressing any of the non-highlighted area immediately closes the playback UI (and keeps the video paused until the user manually presses play)
3. Make it so pressing the display brings the playback UI up (without automatically triggering any of the buttons; the user should have to make a conscious decision to press them) 

**Additional context**
I think changing this would be a big improvement of the UX of the app, and make it feel more premium and similar other video streaming apps.

**Comment by aubertlenno** [comment]
I agree to this, need improvement for better experience. Also adding double tap to rewind and fast-forward would be great.

**Comment by TRtomasz** [comment]
fixed in new mobile app that is available in beta channel
